src/app/components/Persist/MigrateV0StateForm.tsx

```diff
@@ -155,14 +156,14 @@ export function MigrateV0StateForm() {
             />
           </FormField>
           {migratingV0State.invalidPrivateKeys.length > 0 ? (
-            <Button
+            <CountdownButton
```

Comment:
Do we have multiple submit buttons to handle different labels?

Reply:
yeah - ideally these would trigger different handlers too, but we are using a single Form onSubmit. And could be styled differently, but invalid keys are should be so rare that it doesn't matter

Reply:
so they are incidentally only different in labels